app/startup.go

```diff
@@ -70,7 +70,12 @@ func (app *App) startup(ctx context.Context) error {
 
 	app.initStartup(ctx, "Startup.Slack", app.initSlack)
 	app.notificationManager.RegisterSender(notification.DestTypeUserEmail, "smtp", email.NewSender(ctx))
-	app.notificationManager.RegisterSender(notification.DestTypeUserWebhook, "webhook", webhook.NewSender(ctx))
+	if expflag.ContextHas(ctx, expflag.WebhookNC) {
+		app.notificationManager.RegisterSender(notification.DestTypeUserWebhook, "userWebhook", webhook.NewSender(ctx))
+		app.notificationManager.RegisterSender(notification.DestTypeWebhook, "webhook", webhook.NewSender(ctx))
+	} else {
+		app.notificationManager.RegisterSender(notification.DestTypeUserWebhook, "webhook", webhook.NewSender(ctx))
```

Comment:
We shouldn't change the ID based on the flag since these are used for logging. Update the user webhook to `user-webhook` always, and then conditionally load the new one.